tfjs-backend-webgpu/src/kernels/BatchMatMul_impl.ts

```diff
@@ -98,8 +98,8 @@ export function batchMatMulImpl({
 
   const useVec4 = a.shape[2] % 4 === 0 && b.shape[2] % 4 === 0 && !transposeA &&
       !transposeB && outerShapeB >= 32;
-  let program: MatMulPackedProgram|MatMulPackedVec4Program
-      |MatMulSmallOutputSizeProgram;
+  let program: MatMulPackedProgram|MatMulPackedVec4Program|
+      MatMulSmallOutputSizeProgram;
```

Comment:
nit: `let program: WebGPUProgram;`
Then, in line 136, you can directly use `if (program.useWgsl)`.

Reply:
done